charts/jenkins/templates/jenkins-controller-statefulset.yaml

```diff
@@ -70,6 +70,9 @@ spec:
       {{- if .Values.controller.priorityClassName }}
       priorityClassName: {{ .Values.controller.priorityClassName }}
       {{- end }}
+      {{- if .Values.controller.shareProcessNamespace }}
+      shareProcessNamespace: true
```

Comment:
why do you need this for certificates? wouldn't you just mount it as a volume and add the tls cert to the keystore?

Reply:
Our certificates are managed by an external instance of hashicorp vault. It generates the certificates via a sidecar container that it sticks inside the same pod as jenkins. `shareProcessNamespace` allows this sidecar container to trigger config updates / restarts after it (for example) rotates certs